app/lib/features/segment_deletion_config.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module Features
+  class SegmentDeletionConfig < Config; end
```

Comment:
Do you need this?

Reply:
I think that I do because we have different types of features (right now `AccountDeletionConfig` and `SegmentDeletionConfig`) and we need to load them by type (look at the initializer) and fetch them where we use them by type... and even stub them by type. Almost the whole PR is based on that 🤔 check `SegmentDeletionConfig` and you will see 15 results so far 😄

Reply:
sounds good to me.